packages/@storefront/flux-capacitor/src/core/configuration.ts

```diff
@@ -214,6 +214,10 @@ namespace Configuration {
      * enable updating query when hovering over sayt elements
      */
     hoverAutoFill?: boolean;
+    /**
+     * specify optional debounce value in milliseconds
+     */
+    debounce?: number;
```

Comment:
Q: Which debounce value takes precedence? Let's say a user sets a value in the configuration and directly invokes the debounce function through the utils, will there be some sort of clash.

Reply:
**TL;DR**
A. The presence or absence of the `debounce` configuration value will have no effect on an implementations use of `utils.debounce()`.

A: This file specifies that a value for `debounce` *may* be provided as part of the StoreFront configuration object. If the value _is_ provided, then the `Autocomplete` component will use it ti debounce requests for new products. However, this value is not automatically applied to all invocations of `utils.debounce()`.

Reply:
Sounds good, resolving this issue.